### ISSUE TITLE
In check mode, I copied (high...

### ISSUE THREAD

**Issue by door43app** [unfoldingWord-dev/ts-desktop#539]
Notes
======
In check mode, I copied (highlight & Ctrl C) and the source text and then pasted (Ctrl-V) it into the target. Clicked on the checkmark and then marked as done. Copied text disappeared and other strange things happened; i.e. missing source and target verses, wierd characters in Notes

Environment
======
Environment Key | Value
:--: | :--:
Version |6.0.0
Operating System | Windows_NT
Platform | win32
Release | 10.0.10586
Architecture | x64

Log History
======
```javascript
1/4/2016 9:28:24 AM I/bootstrap.js:258:28: Logs are being written to C:\Users\Chris\AppData\Local\translationstudio\log.txt
1/4/2016 9:28:28 AM I/bootstrap.js:92:31: Starting GUI
3/17/2016 9:52:00 AM I/bootstrap.js:258:28: Logs are being written to C:\Users\Chris\AppData\Local\translationstudio\log.txt
3/17/2016 9:52:01 AM I/bootstrap.js:92:31: Starting GUI
3/21/2016 11:01:37 AM I/bootstrap.js:258:28: Logs are being written to C:\Users\Chris\AppData\Local\translationstudio\log.txt
3/21/2016 11:01:37 AM I/bootstrap.js:92:31: Starting GUI
3/21/2016 11:03:52 AM I/bootstrap.js:258:28: Logs are being written to C:\Users\Chris\AppData\Local\translationstudio\log.txt
3/21/2016 11:03:52 AM I/bootstrap.js:92:31: Starting GUI
3/21/2016 11:05:04 AM I/bootstrap.js:258:28: Logs are being written to C:\Users\Chris\AppData\Local\translationstudio\log.txt
3/21/2016 11:05:04 AM I/bootstrap.js:92:31: Starting GUI
3/21/2016 11:12:42 AM I/bootstrap.js:258:28: Logs are being written to C:\Users\Chris\AppData\Local\translationstudio\log.txt
3/21/2016 11:12:42 AM I/bootstrap.js:92:31: Starting GUI
3/21/2016 11:14:15 AM I/bootstrap.js:258:28: Logs are being written to C:\Users\Chris\AppData\Local\translationstudio\log.txt
3/21/2016 11:14:15 AM I/bootstrap.js:92:31: Starting GUI

```

**Comment by cckozie** [comment]
Attached is a file with the text I copied from the programmers console
[console.txt](https://github.com/unfoldingWord-dev/ts-desktop/files/215417/console.txt)

**Comment by cckozie** [comment]
Here is the project file
[en-x-demo1_mat_text_ulb.zip](https://github.com/unfoldingWord-dev/ts-desktop/files/215432/en-x-demo1_mat_text_ulb.zip)

**Comment by jag3773** [comment]
A few notes, 01/12.txt is the chunk that is in question.  Check the history on that file to see what happened.  The initial commit contains the data.

**Comment by cckozie** [comment]
In review mode I was pounding away on pasting the text into the target and marking the chunk as done. When I saw the missing data problem copied some of the messages from the console into the attached file. I'm not sure that I was able to copy all of the relevant messages. As I was trying to copy and organize them, the app window went blank and the console displayed the message "Inspected target disconnect". At that point I was no longer able to scroll through the console log. The tS icon still appears in the taskbar, but its window is gone.
[console-log.txt](https://github.com/unfoldingWord-dev/ts-desktop/files/217674/console-log.txt)
